src/basic/in-addr-util.h

```diff
@@ -187,8 +177,14 @@ static inline int in_addr_prefix_from_string_auto(const char *p, int *ret_family
 }
 
 static inline size_t FAMILY_ADDRESS_SIZE(int family) {
-        assert(IN_SET(family, AF_INET, AF_INET6));
-        return family == AF_INET6 ? 16 : 4;
+        switch (family) {
```

Comment:
`IN_SET()` can be used in the header, hence the change should not be necessary anymore, no?